FAIMS/FAIMS3
# feat: map location 🗺️

## JIRA Ticket

[BSS-469](https://jira.csiro.au/browse/BSS-469)

## Description

Update the map form fields to show the users live current location as a stylised blue dot (I have also added this to the survey map view)

## Proposed Changes

- Created functions to add the following to the map
    1. Center control component
    2. Current location dot
- Added function to get current location
- Updated survey map view and map form fields to use these

## How to Test

1. Navigate to any survey
2. Check that the centre control and current location dot are functioning correctly
3. Create a new record
4. Click "Map Location" and verify the centre control and current location dot are functioning correctly

## Additional Information

## Checklist
